src/Form/Field/TextField.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Kirby\Form\Field;
+
+use Kirby\Cms\ModelWithContent;
+use Kirby\Form\FieldClass;
+use Kirby\Form\Fields;
+use Kirby\Form\Mixin\Converter;
+use Kirby\Form\Mixin\Counter;
+use Kirby\Form\Mixin\Font;
+use Kirby\Form\Mixin\Maxlength;
+use Kirby\Form\Mixin\Minlength;
+use Kirby\Form\Mixin\Pattern;
+use Kirby\Form\Mixin\Spellcheck;
+
+/**
+ * Main class file of the text field
+ *
+ * @package   Kirby Field
+ * @author    Ahmet Bora <ahmet@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     6.0.0
+ */
+class TextField extends FieldClass
+{
+	use Counter;
+	use Converter;
+	use Font;
+	use Maxlength;
+	use Minlength;
+	use Pattern;
+	use Spellcheck;
+
+	public function __construct(
+		string|null $after = null,
+		bool $autofocus = false,
+		string|null $before = null,
+		mixed $converter = null,
+		bool $counter = true,
+		mixed $default = null,
+		bool $disabled = false,
+		string|null $font = null,
+		string|null $help = null,
+		string|null $icon = null,
+		string|null $label = null,
+		int|null $maxlength = null,
+		int|null $minlength = null,
+		ModelWithContent|null $model = null,
+		string|null $name = null,
+		string|null $pattern = null,
+		array|string|null $placeholder = null,
+		bool $required = false,
+		Fields|null $siblings = null,
+		bool $spellcheck = false,
+		bool $translate = true,
+		array|null $when = null,
+		string|null $width = null,
+		mixed $value = null
```

Comment:
Now, the `value` argument will return `null` as a fallback. The `FieldClass::__construct` only fills if the value parameter is passed. This seems like it might cause a break.

https://github.com/getkirby/kirby/blob/3407c4f70b2df37085dd025adf6c0a097b3189f7/src/Form/FieldClass.php#L63-L65

Reply:
I'm not sure I fully understand. You mean we should also check for a null value here?

Reply:
No. Before we doesn't null check but key exists check. So if `$params` doesn't have a value prop, it doesn't touch the value. But with this refactor, `$value` default value is `null`. The class will set the value always `null` if it doesn't passed.

Reply:
@bastianallgeier @distantnative Do you think that's not an issue?